tideways/config.php-profiler.php

```diff
@@ -31,8 +42,8 @@
         return true;
     },
     'profiler.replace_url' => function($uri) {
-        $uri = str_replace('?enable-tideways', '', $uri);
-        $uri = str_replace('%3Fenable-tideways', '', $uri);
+        $uri = str_replace('?enable-tideways=true', '', $uri);
+        $uri = str_replace('%3Fenable-tideways=true', '', $uri);
```

Comment:
users may use `=1` instead or not add an `=` at all, we should be more comprehensive in this check, it's a shame `remove_query_arg` isn't available